src/Integration.Vsix/SonarLintTagger/IIssueMarkerFactory.cs

```diff
@@ -21,10 +21,10 @@
 using Microsoft.VisualStudio.Text;
 using SonarLint.VisualStudio.Core.Analysis;
 
-namespace SonarLint.VisualStudio.Integration.Vsix.Helpers
+namespace SonarLint.VisualStudio.Integration.Vsix
```

Comment:
These files were in the `Helper` subfolder, but they're only used by the tagger.